todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/data/source/local/TasksLocalDataSource.kt

```diff
@@ -94,7 +94,10 @@ class TasksLocalDataSource private constructor(context: Context) : TasksDataSour
             close()
         }
         db.close()
-        task?.let { callback.onTaskLoaded(it) } ?: callback.onDataNotAvailable()
+        if(task!=null){
```

Comment:
Mind correcting the formatting? `if(...){` should be `if (...) {` and `}else` should be `} else`.